src/axom/quest/IntersectionShaper.hpp

```diff
@@ -724,14 +653,14 @@ class IntersectionShaper : public Shaper
       NE,
       AXOM_LAMBDA(axom::IndexType i) {
         // Set each hexahedral element vertices
-        local_hexes[i] = PolyhedronType();
+        local_hexes[i] = HexahedronType();
```

Comment:
Should we also update the `hex_volume` kernel below to use `Hexahedron::volume()` instead of explicitly summing up the tet volumes?